### DISCUSSION TITLE
Release 1.3

### DISCUSSION THREAD

**Discussion by Jagailo** [Jagailo/YetAnotherKeyDisplayer#15] [Announcements]
![image](https://user-images.githubusercontent.com/10468120/162611442-1435c36d-d6e9-4d8d-8c2a-9d10ed5a7e40.png)

# Additions
 * Added key press delay option (only in window mode) (issues #6 and #9).
 * Added tracking of pressing the middle mouse button (issue #12).
 * Added transfer of settings to a new version when installing the application in the same directory.

<hr /><em>This discussion was created from the release <a href='https://github.com/Jagailo/YetAnotherKeyDisplayer/releases/tag/1.3'>Release 1.3</a>.</em>

**Comment by milan93bg** [comment]
Thank you, almost perfect software for keys showup. Just one thing missing and that is minimize to tray button. This would be awesome feature.
![Screenshot_1](https://user-images.githubusercontent.com/42821078/194849217-ca00508a-014f-4ae0-ab15-e24e5e20499e.png)

**Reply by Jagailo** [reply]
Hello, this feature appeared in version [1.4](https://github.com/Jagailo/YetAnotherKeyDisplayer/releases).
